experiments/impttest.py

```diff
@@ -105,7 +105,7 @@ def __init__(self):
         rxctrfreq = katscan_slice["txfreq"]
         txctrfreq = rxctrfreq
         super(ImptTest, self).__init__(cpid, txctrfreq=txctrfreq, rxctrfreq=rxctrfreq, decimation_scheme=create_test_scheme_9(),
-            comment_string="Reimer IMPT test run july 3rd 2019")
+            comment_string="Reimer IMPT Experiment")
```

Comment:
in line above, specified decimation scheme does not need to be here, default is now = the test scheme 9.